src/utilities/__tests__/getBaseName.test.ts

```diff
@@ -10,4 +10,13 @@ describe('utilities/getBaseName', () => {
     expect(getBaseName('/beta/insights/foo/bar/baz')).toEqual('/beta/insights/foo');
     expect(getBaseName('/beta/test/fff/bar/baz')).toEqual('/beta/test/fff');
   });
+
+  it('should find the right base name not on Beta ', () => {
```

Comment:
Isn't this redundant with the "should find the right base name on Stable" test above?